scripts/install_environment.sh

```diff
@@ -46,9 +52,14 @@ function install_libvirt() {
         libvirt-devel${SPECIFIC_LIBVIRT_VERSION} \
         libvirt-daemon-kvm${SPECIFIC_LIBVIRT_VERSION} \
         qemu-kvm \
-        libgcrypt \
-        swtpm \
-        swtpm-tools
+        libgcrypt
+
+    if [ "${DISK_ENCRYPTION:-}" == "tpmv2" ]; then
+        echo "Installing swtpm packages..."
+        sudo dnf install -y \
+            swtpm \
+            swtpm-tools
+    fi
```

Comment:
I don't want to setup the environment every time I need to test swtpm locally

Reply:
the problem is that we'll fail much later, meaning more time debugging tpmv2 job (by others) and spending more money until getting to the desired installation timeouts

Reply:
> I don't think we need to setup the environment every time we want to test swtpm locally

We currently have no choice... You don't have swtpm in new environments (except RHEL 8.6)

Reply:
So we can fail the installation only on swtpm test, e.g.

```
    sudo dnf install -y swtpm swtpm-tools || /bin/true

    if [ "${DISK_ENCRYPTION:-}" == "tpmv2" ]; then
        swtpm --version
    fi

```

Reply:
We might can try to it on different approach, I just don't think that we need to add dependency with any env variable on our environment setup process

Reply:
In general, I totally agree with you. This case is just an exception to the rule because of those packages getting removed in a non-compatible way

BTW, seems like we do depend on ``IPXE_BOOT`` to open those ports: https://github.com/openshift/assisted-test-infra/blob/26be34e92cfe9c9d0d7213eb800def457e13038d/scripts/install_environment.sh#L270
we should probably remove that dependency